model-optimizer/extensions/load/tf/loader.py

```diff
@@ -99,4 +98,38 @@ def load(self, graph: Graph):
         remove_control_dependency_inputs(graph)
 
         graph.check_empty_graph('protobuf2nx. It may happen due to problems with loaded model')
+
+
+class TFExtractor(Loader):
+    id = "TFExtractor"
+    enabled = True
+
+    def run_after(self):
+        return [TFLoader]
+
+    def load(self, graph: Graph):
+        update_extractors_with_extensions(tf_op_extractors)
         extract_node_attrs(graph, lambda node: tf_op_extractor(node, check_for_duplicates(tf_op_extractors)))
+
+
+class TFPrivateExtractor(Loader):
+    id = "TFPrivateExtractor"
+    enabled = False
+
+    def run_after(self):
+        return [TFLoader]
+
+    def load(self, graph: Graph):
+        extract_node_attrs(graph, lambda node: tf_op_extractor(node, {}))
+        for node in graph.get_op_nodes():
+            if node.has('pb') and 'shape' in node.pb.attr:
```

Comment:
Why do we extract only these attributes and not others? Please, add comments in the code.

Reply:
Moved into tests repo